science/dcm2niix/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+PortGroup           github 1.0
+
+github.setup        rordenlab dcm2niix 1.0.20181125 v
+version             ${github.version}
+
+categories          science
+maintainers         {eborisch @eborisch} openmaintainer
+
+description         DICOM to NifTI converter
+
+long_description    dcm2niix is a designed to convert neuroimaging data from \
+                    the DICOM format to the NIfTI format
+
+license             BSD MIT
+platforms           darwin
+
+checksums \
+    rmd160  518b52a35c422511f0f5f73154b14dcaddb5f033 \
+    sha256  d766c345a41954b21d4dc1d90c2c5ddc6425e36228e6d9d8abccd03fc5427c06 \
+    size    312848
+
+depends_build-append    port:git \
+                        port:py27-sphinx
```

Comment:
I assume that upstream has not adapted to python 3.7?

Reply:
Looks like it will work just fine with 3.7; will update.